client/src/components/Tabs/Tabs.js

```diff
@@ -154,20 +154,21 @@ Tabs.defaultProps = {
 export { Tabs as Component };
 
 function mapStateToProps(state, ownProps) {
-  const id = ownProps.id;
-  const field = (state.tabs.fields[id])
-    ? state.tabs.fields[id]
+  const fieldID = `${ownProps.formid}__${ownProps.id}`;
```

Comment:
Let's move this to a helper function `createFieldID(ownProps)` to reduce repetition.